src/main/scala/code/service/Color.scala

```diff
@@ -0,0 +1,31 @@
+package code.service
+
+import java.util.Random
+
+case class Color(red: Int, green: Int, blue: Int, alpha: Int = 1) {
+  val isDark: Boolean = (((red * 299) + (green * 587) + (blue * 114)) / 1000) < 128
+
+  override def toString: String = s"($red,$green,$blue,$alpha)"
+}
+
+object Color {
+  val transparent: Color = Color(0, 0, 0, 0)
+
+  def get(taskName: String, projectsDisplayName: String, active: Boolean): Color = {
+    val random = new Random((taskName.trim + projectsDisplayName.trim).hashCode)
+    Color(
+      red = if (active) random.nextInt(255) else 255,
+      green = if (active) random.nextInt(255) else 255,
+      blue = if (active) random.nextInt(255) else 255,
+      alpha = if (active) 1 else 1
```

Comment:
Here is the [original](https://github.com/dodie/time-admin/blob/master/src/main/scala/code/service/TaskService.scala#L28) color generating method.